### PR TITLE
Add ZipkinReporter

### PR DESCRIPTION
**Pull request by chingor13**
This PR adds a `ZipkinReporter` which using built-in http streaming capabilities to report to a Zipkin (HTTP API)[http://zipkin.io/zipkin-api/#/paths/%252Fspans]. The reporter adapts the collected `TraceSpans` into the span format for Zipkin.

This was tested against a local Zipkin instance running in docker.

### CODE REVIEW

File: tests/unit/Trace/Reporter/ZipkinReporterTest.php [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+namespace OpenCensus\Tests\Unit\Trace\Reporter;
+
+use OpenCensus\Trace\Reporter\ZipkinReporter;
+use OpenCensus\Trace\TraceContext;
+use OpenCensus\Trace\TraceSpan;
+use OpenCensus\Trace\Tracer\TracerInterface;
+use Prophecy\Argument;
+
+/**
+ * @group trace
```
Comment (tmatsuo) [inline tests/unit/Trace/Reporter/ZipkinReporterTest.php]: Is `@group trace` still relevant?

Comment (chingor13) [inline tests/unit/Trace/Reporter/ZipkinReporterTest.php]: For now, they are all in the trace group. OpenCensus does eventually plan to have other metrics and stats reporting in the future.

Comment (tmatsuo) [inline tests/unit/Trace/Reporter/ZipkinReporterTest.php]: LG

**Comment by adriancole** [comment]
Great stuff and thanks! PS if it makes it any easier, very latest zipkin supports a simpler json format. I know that probably for existing sites you may need to retain the v1 format until folks upgrade.

https://github.com/openzipkin/zipkin-api/pull/33

**Pull request merged**